README.md

```diff
@@ -0,0 +1,191 @@
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)  
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+# 📖 Book an appointment | Places <a name="about-project"></a>
+
+> The Book an appointment | Places API will provide functionality to create, read, and destroy data related to places. It will serve as the server-side component responsible for handling requests and managing the database. The React client will interact with this backend to perform actions such as creating new places and reservations, see the place details, and removing places from the system.
+
+[Link to the Front End Repo](https://github.com/dennis-every/book-an-appointment-front-end)
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+    <ul>
+    <li><a href="https://react.dev/">React</a></li>
+    <li><a href="https://redux.js.org/">Redux</a></li>
+  </ul>
+</details>
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+  </details>
+  <details>
+  <summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Data Model**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+> You need the following tools be installed in your computer:
+
+> - [Rails](https://guides.rubyonrails.org/)
+> - [Git](https://www.linode.com/docs/guides/how-to-install-git-on-linux-mac-and-windows/)
+> - [Ruby](https://github.com/microverseinc/curriculum-ruby/blob/main/simple-ruby/articles/ruby_installation_instructions.md)
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+cd my-prefered-folder
+  
+git clone https://github.com/dennis-every/book-an-appointment-api.git
+
+```
+
+### Install
+
+Install this project with:
+
+```sh
+cd book-an-appointment-api
+  
+bundle
+```
+
+### 💾 Database
+
+Create the databases, run migrations, and populate it with seed data:
+
+```sh
+rails db:create
+rails db:migrate
+rails db:seed
+```
+
+### Usage
+
+To run the project run the following commands:
+
+```sh
+rails s
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+
+```sh
+rspec spec --exclude-pattern "spec/integration/api/*_spec.rb"
```

Comment:
[OPTIONAL] 

Should we really have to exclude these tests to have all tests pass? Consider trying to find the root cause of failed tests..